src/css/support.js

```diff
@@ -96,7 +96,11 @@ define([
 			div.removeChild( marginDiv );
 
 			return ret;
-		}
+		},
+		// Support: IE 11
+		// IE 11 has a bug with offsetWidth / offsetHeight / getClientBoundingRect()
+		// in fullscreen mode inside an iframe
```

Comment:
I'd describe what exactly the bug is about and link to the MS Connect bug about it.